.docker/Dockerfile.cli

```diff
@@ -1,16 +1,33 @@
-FROM govcms8lagoon/govcms8
+##
+# `cli` image.
+#
+# The application is built in this image as it contains build tools like
+# composer and npm.
+#
+# The default PaaS build strategy is:
+#
+#  * wipe the /app directory, from the upstream image build (which is currently SaaS oriented).
+#  * prepare and run composer install inside the `cli` container
+#  * copy in the non-composer files
+#
 
+FROM amazeeio/php:7.2-cli-drupal
```

Comment:
Analysis of govcms8lagoon/govcms8 says it's not adding much value and we repeat most of what it does, especially for what is essentially a non-public facing container.